nodepool/files/nodepool.yml

```diff
@@ -113,6 +113,11 @@ providers:
         availability-zones: []
         labels: &provider_pools_main_labels_block
           #MaaS
+          - name: maas-ubuntu-bionic-g1-8
```

Comment:
This is not enough to complete the configuration for adding a new node type. Look for the current use of ``maas-ubuntu-xenial-g1-8`` in the file and replicate those for bionic.

Reply:
Done